content/about/verslagen/17-18/2018-04-19.md

```diff
@@ -0,0 +1,174 @@
+        -   Alumniwerking
+            -   Alumni-event (receptie of alumni-codenight) voor alle sponsors en ereleden (eventueel ereleden).
+        -   Huidige ereleden: mail sturen. Ereleden kunnen storten tot aan de bestuursverkiezingen.
+- Bestuursfuncties '18-'19
+  Momenteel is mogelijk:
+    - **Voorzitter (verplicht)**
+      Het hoofd van het bestuur. Hakt knopen door bij moeilijke beslissingen en zorgt voor een vloeiende werking van het bestuur 
+    - **Penning (verplicht)**
+      De penningmeester houdt alle financiële zaken bij van de organisatie.
+      Deze functie is ook verplicht door de DSA, en kan misschien gecombineerd worden met secretaris, wegens het ook een administratieve taak is.
+    - **Secretaris**
+      De secretaris onderhoudt de administratie van het bestuur en schrijft de verslagen van de bestuursvergaderingen. Ook reserveert de secretaris de evenementen van Zeus en noteert ze in het DSA-controlepaneel. Deze taak kan door PR opgenomen worden.
+    - **Project**
+      De projectverantwoordelijke verzorgt de projecten in Zeus en zorgt voor een goede vooruitgang. Ook zorgt de projectverantwoordelijke voor de definitie van een goed development proces en onderhoudt de repos
+    - **Sysadmin**
+      De sysadmin verzorgt de servers, zij het lokaal, zij het in de cloud. Deze moeten altijd up to date gehouden worden zodat er geen security issues op kunnen komen.
+    - **PR**
+      De PR verantwoordelijke zorgt voor blogposts, communicatie met externen en onze social media. De PR zal dus ook foto's en dergelijke online zetten en het engagement met niet-Zeusleden verhogen
+    - **HR**
+      SHFEERBEHEER. Zorgt voor de algemene sfeer in de kelder en ledenwerking. Probeert actief leden aan te werven. Kennisoverdracht tussen events enzovoorts.
+      
+  Bestuursfuncties die we niet inrichten:
+    - **Vicevoorzitter (verplicht)**
+      Right hand (wo)man van de voorzitter. Wordt commander-in-chief indien de voorzitter niet beschikbaar is.
+      Deze functie is verplicht door DSA, maar in de realiteit is de vice gewoon secretaris met een fancy titel. In principe kan eender welk bestuurslid de vicevoorzitter zijn, ongeacht hun feitelijke rol.
+      Het voorstel is om deze rol intern aan te duiden onder de verkozen bestuursleden.
+    - **Event**
+      In plaats van één persoon constant te 'belasten' met het organiseren van alle events zouden we blijven bij de formule van dit jaar: nl. events in het begin van het semester te bespreken en een verantwoordelijke per event aan te duiden.
+    
+    Discussie
+  - HR samenvoegen met Vice + Vice beter omschrijven
+  - Projectverantwoordelijke is meer projecten overzien in tegenstelling tot coden.
```

Comment:
Dit is toch geen discussiepunt? We zijn het er allen toch over eens dat een projectverantwoordelijke een organisatorische functie heeft en geen codemonkey is?

We hebben een discussie gehad over in hoeverre een projectverantwoordelijke een voortrekkersrol heeft (cfr MOZAIC dit jaar), en de werklast die daar bij komt kijken.

Reply:
Dat is correct, ik zal het anders verwoorden.